README.md

```diff
@@ -107,6 +108,13 @@ You can deploy this project using: GitHub but if you want to deploy make sure yo
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Contact section**
+- [ ] **Navigation functionality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 🤝 Contributing <a name="contributing"></a>
 
```

Comment:
- [ ] OPTIONAL
MIT license link on _line 140_ of your ReadMe is broken, kindly endeavor to fix it.

Reply:
The MIT license link is now working, thanks very much for your time.